src/pagination.scss

```diff
@@ -9,23 +9,16 @@
 $block: #{$fd-namespace}-pagination;
 
 .#{$block} {
-  $fd-pagination-link-color: $fd-color--action !default;
-  $fd-pagination-link-color--active: fd-color("text", 2) !default;
-  $fd-pagination-total-color: fd-color("text", 2) !default;
-
-  --fd-pagination-link-color: var(--fd-color-action-1);
-  --fd-pagination-link-color-active: var(--fd-color-text-2);
-  --fd-pagination-total-color: var(--fd-color-text-2);
-
   @include fd-reset();
-  @include fd-type("0");
+
+  font-size: var(--sapFontSize);
 
   &__total {
     @include fd-reset();
-    @include fd-type("0");
-    @include fd-var-color("color", $fd-pagination-total-color, --fd-pagination-total-color);
 
-    margin-right: fd-space("xxs");
+    font-size: var(--sapFontSize);
+    color: var(--sapContent_LabelColor);
+    margin-right: 0.5rem;
```

Comment:
You need RTL for this. It should become 0 and margin-left: 0.5rem. It will be better to move it to a variable